perfkitbenchmarker/linux_packages/cassandra.py

```diff
@@ -193,7 +193,7 @@ def StartCluster(seed_vm, vms):
   logging.info('Starting seed VM %s', seed_vm)
   Start(seed_vm)
   logging.info('Waiting %ds for seed to start', NODE_START_SLEEP)
-  time.sleep(NODE_START_SLEEP)
+  time.sleep(NODE_START_SLEEP * 2)
```

Comment:
Why 2 here? Looks like it retries anyway, and this doesn't match the log in the previous line.